recipe/meta.yaml

```diff
@@ -10,13 +10,6 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    ###################################################################
-    # Apply test fixture fix from upstream.                           #
-    #                                                                 #
-    # xref: https://github.com/zarr-developers/zarr-python/pull/1364  #
-    ###################################################################
-    - patches/1364.patch
```

Comment:
As PR ( https://github.com/zarr-developers/zarr-python/pull/1364 ) was included in `2.15.0`, this patch (understandably) no longer applies and further is no longer needed

Reply:
_facepalm_ Makes sense. Needed more sleep. 😉 Thanks!